cibuildwheel/windows.py

```diff
@@ -31,26 +37,51 @@ def get_python_path(config):
             arch = '-x64' if config.arch == '64' else ''
         )
 
+        
 
 def get_python_configurations(build_selector):
-    PythonConfiguration = namedtuple('PythonConfiguration', ['version', 'arch', 'identifier', 'path'])
+
+    bace_choco_args = "--no-progress --force -y --allowmultiple --override".split() 
+    instal_args = "'/quiet  InstallAllUsers=1 TargetDir={}'"
+    
+    class PythonConfiguration(object):
```

Comment:
Just a though: could we keep this as a simple `namedtuple`, and format/create the full args when we need them.

We could even just make a simple function `get_choco_args` that takes the `version` and `arch` and returns all the arguments? Then we don't need to change the current `PythonConfiguration`.